messaging/kafka_topics.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# (c) 2015, James Bowes <jbowes@repl.ca>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = """
+---
+module: kafka_topics
+short_description: Create and modify Kafka topics.
+description:
+  - Create and modify I(Kafka) topics.
+notes:
+  - If you have multiple topics to modify, using the I(topics) option is much
+    faster, as it can execute C(kafka-topics.sh) fewer times.
+requirements:
+  - C(kafka-topics.sh) must be installed on the host.
+author: James Bowes
+options:
```

Comment:
Yeah! I'll look in to what happens if you try and delete on an older kafka that doesn't support it. If things seem safe, I'll add it.